### DISCUSSION TITLE
How to let bot send message after a few second?

### DISCUSSION THREAD

**Discussion by breaker84** [LiveHelperChat/livehelperchat#1926] [General]
Hello there, is there any chance that the bot will send the following message after a few seconds / delay? Af current, if I set a few respond my bot, all will appear at one time. I was hoping that we can delay the response in sec. Example

Human : How to login page A?
Bot : 1st, you need to know your id.
..... (delay 2 sec) <---this delay 
Bot : Then enter the id in the page login.

**Comment by remdex** [comment]
Use response type `Send typing`
![image](https://user-images.githubusercontent.com/1146085/227190013-3ae4c375-ff97-4611-9128-2f258b078fd4.png)

Or if delay required at system level use
![image](https://user-images.githubusercontent.com/1146085/227190206-d57d486e-bc50-4838-8566-04c2b9f876ae.png)

**Comment by breaker84** [comment]
Nice one...
Love this app...
Thanks remdex
